docs/glossary.rst

```diff
@@ -1,2 +1,58 @@
 Glossary
 ========
+
+.. glossary::
+
+    :class:`~pulpcore.plugin.models.Artifact`
+        A file that belongs to a :term:`content unit<content>`.
+
+    :class:`~pulpcore.plugin.models.Content`
```

Comment:
I am not sure what is ideal and am open to suggestions, but frequently the term "content unit" is linked out to be defined and it lands here at "Content" which you can see from the url "glossary.html#term-content", but as the glossary page is so short, instead of going straight to "Content" we are just shown the whole glossary page which can be confusing when there is no entry for "content unit".  We define Content by saying when singular it is "content unit" but perhaps we need an actual "content unit" entry that simply says "The singular form of Content" and links here?  On the concepts.html page, it says "This document favors clarity and accuracy over ease of reading." and I think in favor of clarity, we need to define content unit given the frequency we link to the glossary from it.